---
layout: article
titles:
  # @start locale config
  en      : &EN       CV
  en-GB   : *EN
  en-US   : *EN
  en-CA   : *EN
  en-AU   : *EN
  zh-Hans : &ZH_HANS  关于
  zh      : *ZH_HANS
  zh-CN   : *ZH_HANS
  zh-SG   : *ZH_HANS
  zh-Hant : &ZH_HANT  關於
  zh-TW   : *ZH_HANT
  zh-HK   : *ZH_HANT
  ko      : &KO       소개
  ko-KR   : *KO
  fr      : &FR       CV
  fr-BE   : *FR
  fr-CA   : *FR
  fr-CH   : *FR
  fr-FR   : *FR
  fr-LU   : *FR
  # @end locale config
key: page-about
permalink: /CV/
aside:
    toc: true
---

## Education
- **PhD** in Organismic and Evolutionary Biology, Harvard University, 2015
    - Dissertation: "*Evolutionary dynamics of a multiple-ploidy system in Arabidopsis arenosa*"
    - Advisors: Kirsten Bomblies (primary), Nancy Kleckner (secondary)
    - PhD Committee: John Wakeley, David Reich, Hopi Hoekstra
- **BS** in Plant Biology, University of Minnesota -- Twin Cities, 2008
    - *Honors* and *High Distinction*

## Professional Exp
- **Senior Biomedical Data Scientist**, Princeton University (2020-present)
    - Teach workshops, collaborate with faculty, develop software
- **Senior Bioinformatics Scientist III**, Harvard University (2018-2020)
    - Teach workshops, collaborate with faculty, help students analyze large sequencing datasets using high-performance computing
- **Consultant**, Day Zero Diagnostics (2018)
    - Bacterial genomics, transmission dynamics within Massachussetts General Hospital

## Research Exp
- Ruth L. Kirschstein **Postdoctoral Fellow**, Harvard T.H. Chan School of Public Health (2015-2018)
    - Advisors: Bill Hanage, Marc Lipsitch, Yonatan Grad
- **Visiting Scientist**, University of Helsinki (2016)
    - Jukka Corander's group: Bayesian optimization, approximate bayesian computation, machine learning
- **Fulbright Fellow**, University of Oulu (2009)
    - Advisor: Outi Savolainen

## Skills
- **Programming**: proficient in Python/Snakemake, R, Unix/Bash, Perl; familiar with C++
- **Bioinformatics**: variant calling, genome assembly, metagenomics, high-performance and cloud computing 
- **Teaching and mentoring**: passionate about student development, patient with beginners
- **Written and oral communication**: 16 scientific publications, 4 successful grant/fellowship applications, 3 invited talks
- **Able to work independently and as part of a team**: creative thinker, enjoy collaborations and helping trainees

## Academic Awards
- **Postdoctoral Awards** (Harvard T.H. Chan School of Public Health)
    - NIH F32 Postdoctoral Fellowship (2016-2018)
- **Graduate Awards** (Harvard University)
    - NSF Doctoral Dissertation Improvement Grant (2012)
    - 2X Certificate of Distinction in Teaching (2010, 2012)
    - NSF Graduate Research Fellowship (2010-2015)
    - Herchel Smith Graduate Fellowship (2009-2011)
    - James Mill Peirce Fellowship (2009)
- **Undergraduate Awards** (University of Minnesota -- Twin Cities)
    - Fulbright Full Grant (2008-2009)
        - Advised by Outi Savolainen at University of Oulu, Finland
    - Merck Index Award (2007)
        - best student in organic chemistry series
    - Prentice Hall Book Prize (2006)
        - highest grade in organic chemistry II

## Teaching Exp.
- As [Senior Biomedical Data Scientist](#professional-exp)
    - Introduction to Machine Learning (2022)
    - Data Vizualization in Python (2022)
    - Best Practices in Python Packaging (2021)
    - Introduction to R (2021)
- As [Senior Bioinformatics Scientist](#professional-exp)
    - Introduction to R with Tidyverse (2019-2020)
    - Read mapping and variant valling (2019-2020)
- As Ruth L. Kirschstein [Postdoctoral Fellow](#academic-exp)
    - Mentored Master's Student (2016-2017)
    - Dept. of Epidemiology summer program (2016)
- As [Graduate](#education) teaching fellow
    - Genetics and Genomics, OEB 50 (2012)
    - Coalescent Theory, OEB 252 (2010)
- As [Undergraduate](#education) teaching assistant
    - Biochemistry
    - Organic Chemistry
    - General Botany

## Publications

- Peer-reviewed publications

    - Xiao A, Wu F, Bushman M, Zhang J, Imakaev M, Chai PR, Duvallet C, Endo N, Erickson TB, Armas F, **Arnold B**, Chen H, Chandra F, Ghaeli N, Gu X, Hanage WP, Lee WL, Matus M, McElroy KA, Moniz K, Rhode SF, Thompson J, and EJ Alm (2022). Metrics to relate COVID-19 wastewater data to clinical testing dynamics. *Water Research* 212 (2022) 118070.

    - **Arnold B**, Huang IT, and WP Hanage (2022). Horizontal gene transfer and adaptive evolution in bacteria. *Nature Reviews Microbiology* 20, 206-218.

    - Campbell-Staton SC**†**, **Arnold B†**, Gonçalves D, Granli P, Poole J, Long RA, and Pringle RM (2021). Ivory poaching and the rapid evolution of tusklessness in African elephants. *Science* 374, 483-487.
        - **† these authors contributed equally to this work**

    - Cai L, **Arnold B**,Xi Z, Khost D, Patel N, Hartmann C, Manickam S, Sasirat S, Nikolov LA, Mathews S, Sackton T, and CC Davis (2021). Deeply altered genome architecture in the endoparasitic flowering plant *Sapria himalayana* Griff. (Rafflesiaceae). *Current Biology*: 31 (5), 1002-1011.e9.

    - Azarian T, Martinez P, **Arnold B**, Qiu X, Grant L, Corander J, Fraser C, Croucher N, Hammitt L, Reid R, Santosham M, Weatherholtz R, Bentley S, O’Brien K, Lipsitch M, and WP Hanage (2020). Frequency-dependent selection can forecast evolution in Streptococcus pneumoniae. *PLoS Biology*: https://doi.org/10.1371/journal.pbio.3000878. 

    - **Arnold B**, Sohail M, Wadsworth C, Corander J, Hanage WP, Sunyaev S, and Y Grad (2019). Fine-scale haplotype structure reveals strong signatures of positive selection in a recombining bacterial pathogen. *Molecular Biology and Evolution* 37(2):417-428.

    - Pensar J, Puranen S, **Arnold B**, MacAlasdair N, Kuronen J, Tonkin-Hill G, Pesonen M, Xu Y, Sipola A, Sánchez-Busó  L, Lees J, Chewapreecha C, Bentley S, Harris S, Parkhill J, Croucher N, and J Corander (2019). Genome-wide epistasis and co-selection study using mutual information. *Nucleic Acids Research* 47(18):e112-e112.

    - Monnahan P, Kolář F, Baduel  P, Sailer  C, Koch J, Horvath R, Laenen B, Schmickl  R, Paajanen  P, Šrámková  G, Bohutínská  M, **Arnold B**, Weisman  C, Marhold K, Slotte T, Bomblies K, and L Yant (2019). Pervasive population genomic consequences of genome duplication in Arabidopsis arenosa. *Nature Ecology and Evolution* 3(3):457-468

    - Shapiro L, Paulson J, **Arnold B**, Scully E, Zhaxybayeva O, Pierce N, Rocha J, Klepac-Ceraj V, Holton K, and R Kolter (2018). An introduced crop plant is driving diversification of the virulent plant pathogen Erwinia tracheiphila. *mBio* 9:e01307-18.

    - Wadsworth C, **Arnold B**, Sater M, and Y Grad (2018). Azithromycin resistance through interspecific acquisition of an epistasis dependent efflux pump component and transcriptional regulator in Neisseria gonorrhoeae. *mBio* 9:e01419-18

    - Azarian T, Grant L, **Arnold B**, Hammitt L, Reid R, Santosham M, Weatherholtz R, Goklish N, Thompson C, Bentley S, O’Brien K, Hanage WP, and M Lipsitch (2018). The impact of serotype-specific vaccination on phylodynamic parameters of Streptococcus pneumoniae and the pneumococcal pan-genome. *PLoS Pathogens*: https://doi.org/10.1371/journal.ppat.1006966.

    - **Arnold B**, Gutmann M, Grad Y, Sheppard S, Corander J, Lipsitch M, and WP Hanage (2018). Weak epistasis may drive adaptation in recombining bacteria. *Genetics* 208(3):1247-1260.

    - Corander J, Fraser C, Gutmann M, **Arnold B**, Hanage WP, Bentley S, Lipsitch M, and N Croucher (2017). Frequency-dependent selection in vaccine-associated pheumococcal population dynamics. *Nature ecology & evolution* 1:1950-1960.

    - **Arnold B**, Lahner B, DaCosta J, Weisman C, Hollister J, Salt D, Bomblies K, and L Yant (2016). Borrowed alleles and convergence in serpentine adaptation. *Proceedings of the National Academy of Sciences of the USA*, 113(29): 8320-8325.

    - Baduel P, **Arnold B**, Weisman C, Hunter B, and K Bomblies (2016). Habitat-associated life history and stress-tolerance variation in Arabidopsis arenosa. *Plant physiology*, 171(1): 437-451.

    - **Arnold B**, Kim S, and K Bomblies. (2015). Single origin of autotetraploid Arabidopsis arenosa followed by interploidy admixture. *Molecular Biology and Evolution*, 32(6):1382-1395.

    - Wright K, **Arnold B**, Xue K, Šurinová M, O'Connell J, and K Bomblies, (2014). Selection on meiosis genes in diploid and tetraploid Arabidopsis arenosa. *Molecular Biology and Evolution*, 32(4): 944-955.

    - **Arnold B**, Corbett-Detig R, Hartl D, and K Bomblies (2013). RADseq underestimates diversity and introduces genealogical biases due to nonrandom haplotype sampling. *Molecular Ecology*, Vol 22: 3179-3190.

    - Yant L, Hollister JD, Wright K, **Arnold B**, Higgins JD, Franklin F, and K Bomblies, (2013). Meiotic adaptation to genome duplication in Arabidopsis arenosa. *Current Biology*. 23, pp. 2151-2156.

    - Hollister J, **Arnold B**, Svedin E, Xue K, Dilkes B, and K Bomblies, (2012). Genetic adaptation associated with genome-doubling in autotetraploid Arabidopsis arenosa. *PLoS Genetics*, 8(12): e1003093

    - **Arnold B**, Bomblies K, and J Wakeley (2012). Extending coalescent theory to autotetraploids. *Genetics* 192(1):195-204.

